### PR TITLE
Optimize CI for testing new fuzzer addition.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
This removes the ability to test adding new benchmarks easily in CI. That should be replaced with something in gcb, since mostly maintainers do this.

Related:
https://github.com/google/fuzzbench/issues/1556
https://github.com/google/fuzzbench/issues/1590

### CODE REVIEW

File: .github/workflows/build_and_test_run_fuzzer_benchmarks.py [hunk 1]
```diff
@@ -85,16 +83,16 @@ def run_command(command):
...
 
 
-def make_builds(benchmarks, fuzzer):
-    """Use make to test the fuzzer on each benchmark in |benchmarks|."""
-    fuzzer_benchmark_pairs = builder.get_fuzzer_benchmark_pairs([fuzzer],
-                                                                benchmarks)
+def make_builds(benchmark, fuzzers):
+    """Use make to test the |benchmark| on each fuzzer in |fuzzers|."""
+    fuzzer_benchmark_pairs = builder.get_fuzzer_benchmark_pairs(
+        fuzzers, [benchmark])
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply .github/workflows/build_and_test_run_fuzzer_benchmarks.py:89]: I disagree. Maybe I'm missing something but get_fuzzer_benchmark_pairs is used by other code and must accept lists.

[the remark this answers is not in the document]
Comment (Alan32Liu) [reply .github/workflows/build_and_test_run_fuzzer_benchmarks.py:89]: I see, I did not realize it was also used by [`get_fuzzer_benchmark_pairs`](https://github.com/google/fuzzbench/blob/5e37b392529208dbcfcda8699ee091dbcaab1020/experiment/build/builder.py#L187).
Thanks for the reminder!

**Comment by jonathanmetzman** [comment]
I have no idea where the CI failure is coming from. My first guess is it is unrelated.

**Comment by Alan32Liu** [comment]
Shall we merge this? @jonathanmetzman

**Comment by jonathanmetzman** [comment]
Done!

**Pull request merged**